website/docs/getting-started/setup/instance-configuration/authentication/openid-connect-oidc/aws-cognito.md

```diff
@@ -0,0 +1,83 @@
+
+    ![](/img/Step2-AWS_Cognito.png)
+
+4. **Step 3 - configure sign-up experience:** keep the default configurations and click *Next*.
+5. **Step 4 -Configure message delivery:** you can choose either of the options - *Send email with Amazon SES* or *Send email with Cognito*. Please pick *Send email with Cognito* for this setup.
+6.  **Step 5 - Integrate your app:**
+    1. Enter a user pool name.
+    2. Under *Initial app client*, add an app client name and select **Generate a client secret** under Client secret section.
+
+    ![](/img/AWS-cognito_App_client.png)
+
+7. **Step 6 - Review Configurations:** review all the selected configurations and proceed once done.
+8. Click the user pool you've created, go to the integrations tab and follow the steps below 
+    1. In the domains section, click on the actions dropdown and select **Create a Cognito domain**. You can also select **Create a custom domain** to add a domain that you own.
+
+    ![](/img/AWS-cognito_create-cognito-domain.png)
+
+    2. Enter a domain prefix in the **Create Cognito domain** screen and click on **Create Cognito domain** button.
+
+        ![](/img/AWS-cognito_create_domain.png)
+
+9. On the user pool home, scroll down to the **App client list** and open the App Client that you created.
+    1. Copy the client ID and client secret that will be added to the OIDC configurations in Appsmith.
+
+    ![](/img/AWS-cognito_client-creds.png)
+
+    2. Scroll down to the Hosted UI section and click Edit. Click the Add Callback URL button and enter Redirect copied from the OIDC window in Appsmith’s Admin Settings.
+
+## Configure AWS Cognito with Appsmith
+
+On your Appsmith instance, go to *Admin Settings> Authentication> OIDC*, click Enable and follow the steps below:
+
+1. Copy the redirect URL and add it as a Callback URL under the Hosted UI section of your AWS Cognito User pool.
+
+![](/img/AWS-cognito_callback-url.png)
+
+2. Add the client ID and client secret key copied from the App Client in your user pool to the respective input boxes.
+3. Add the remaining configurations in the format shown below:
+
+    | OIDC configuration field in Appsmith | AWS Cognito URL |
+    | -------------------------------------|----------------------- |
+    | Authorization URL                    | `https://<your_user-pool_domain>/oauth2/authorize`|
+    | Token URL                            | `https://<your_user-pool_domain>/oauth2/token` |
+    | User Info URL                        | `https://<your_user-pool_domain>/oauth2/userInfo` |
+    | JWK Set URL                          | `https://cognito-idp.<Region>.amazonaws.com/<userPoolId>/.well-known/jwks.json` | 
+
+    **your_user-pool_domain** - The Cognito domain you created in the above step.
+
+    **Region** - The region's code where your AWS Cognito is set up.
+
+    **userPoolId** - The unique ID of the user pool you've created.
+
+    ![](/img/AWS_Urls.png)
+
+### Complete OIDC setup
```

Comment:
no need of this separate section. The Save & Restart button can be part of the continuation of the above steps